cpu/nrf52/vectors.c

```diff
@@ -34,6 +35,7 @@ void dummy_handler(void) {
 WEAK_DEFAULT void isr_power_clock(void);
 WEAK_DEFAULT void isr_radio(void);
 WEAK_DEFAULT void isr_uart0(void);
+WEAK_DEFAULT void isr_spi0(void);
```

Comment:
Does this CPU really have a new interrupt?

Reply:
See the other isr.